src/components/DelegateStatement/CurrentDelegateStatement.tsx

```diff
@@ -140,8 +170,8 @@ export default function CurrentDelegateStatement() {
     }
   }, [address, reset]);
 
-  if (!isConnected) {
-    return <ResourceNotFound message="Oops! Nothing's here" />;
+  if (!isConnected && !isConnecting) {
+    router.push("/");
   }
```

Comment:
this pushes you back home if you are not connected? I feel like this might be jarring if you are not expecting it.